Dockerfile.nginx

```diff
@@ -1,2 +1,4 @@
 FROM nginx:alpine
-COPY ./docker_entrypoint/nginx/default.conf /etc/nginx/conf.d/default.conf
+COPY ./docker_entrypoint/nginx/default.conf.template /etc/nginx/conf.d/default.conf.template
+CMD DOMAIN_NAMES=$(echo $VIRTUAL_HOST | sed 's/,/ /g') envsubst '$DOMAIN_NAMES' < /etc/nginx/conf.d/default.conf.template \
+    && DOMAIN_NAMES=$(echo $VIRTUAL_HOST | sed 's/,/ /g') envsubst '$DOMAIN_NAMES' < /etc/nginx/conf.d/default.conf.template > /etc/nginx/conf.d/default.conf && nginx -g 'daemon off;'
```

Comment:
consider `cat`ing the resulting file instead of doing the substitution twice (once for output, once for file)